php_phongo.c

```diff
@@ -2445,6 +2445,12 @@ PHP_MINIT_FUNCTION(mongodb)
 
 	/* Initialize libmongoc */
 	mongoc_init();
+
+#ifdef MONGOC_EXPERIMENTAL_FEATURES
```

Comment:
It seems a bit weird to talk about these things in a unrelated pull request.

I'd be more then happy to schedule a call to talk about this. Unfortunately, I cannot do 7am on Monday mornings like has been previously suggested, but normal working hours I can move things around to accommodate.